plugin.onedrive/addon.xml

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.onedrive" name="OneDrive" version="1.1.12" provider-name="Carlos Guzman (cguZZman)">
+<addon id="plugin.onedrive" name="OneDrive" version="1.2.0" provider-name="Carlos Guzman (cguZZman)">
 	<requires>
 		<import addon="xbmc.python" version="2.19.0" />
 	</requires>
 	<extension point="xbmc.python.pluginsource" library="addon.py">
 		<provides>image audio video</provides>
 	</extension>
+	<extension point="xbmc.service" library="service.py" start="login" />
 	<extension point="xbmc.addon.metadata">
 		<platform>all</platform>
 		<summary lang="en">Microsoft OneDrive for KODI</summary>
 		<description lang="en">
 Play all your media from Microsoft OneDrive including Videos, Music and Pictures.
- - Unlimited number of accounts.
+ - Unlimited number of accounts. Personal or Business.
```

Comment:
Not sure if it applies but probably a good idea do use [CR] for line breaks. Not a blocker though.